homeassistant/components/advantage_air/climate.py

```diff
@@ -164,34 +165,34 @@ def __init__(self, instance, ac_key, zone_key):
         )
 
     @property
-    def hvac_mode(self):
+    def hvac_mode(self) -> HVACMode | str | None:
         """Return the current state as HVAC mode."""
         if self._zone["state"] == ADVANTAGE_AIR_STATE_OPEN:
             return HVACMode.HEAT_COOL
         return HVACMode.OFF
 
     @property
-    def current_temperature(self):
+    def current_temperature(self) -> float | None:
         """Return the current temperature."""
         return self._zone["measuredTemp"]
 
     @property
-    def target_temperature(self):
+    def target_temperature(self) -> float | None:
```

Comment:
Can this actually return None? or is it always a float?

Reply:
Fixed